### ISSUE TITLE
GridList a11y

### ISSUE THREAD

**Issue by danielstclair** [MerlinLabs/styled-material-components#176]
The GridList component should have better support for a11y.

Example use case:

I've used the GridList component to render a list that I have.
I go to a screen reader and it does not read the list as expected or tell me that the item that it's reading represents a list.

This is just one use case that I can think of at the moment but some items we should consider are:

- How do we account for different kinds of lists?
- Keyboard navigation.
- Something else??
